src/ducks/myApps/components/Discover.jsx

```diff
@@ -22,7 +32,12 @@ export class Discover extends Component {
           </h3>
           <div className='sto-discover-get-started-list'>
             {!isFetching && apps.map(a => {
```

Comment:
Could we use an  `app` variable here ? I hate one-letter variables, even in simple cases like this one. If I look only line 42 for example I read only `slug={a.slug}` and I do not understant what we are dealing with. What do think ?